src/app/components/FatalErrorHandler/index.tsx

```diff
@@ -3,29 +3,52 @@
  * FatalError
  *
  */
-import { AlertBox } from 'app/components/AlertBox'
 import { selectFatalError } from 'app/state/fatalerror/selectors'
 import copy from 'copy-to-clipboard'
+import styled from 'styled-components'
+import { ResponsiveContext } from 'grommet/es6/contexts/ResponsiveContext'
+import { ThemeContext } from 'styled-components'
 import { Anchor } from 'grommet/es6/components/Anchor'
 import { Box } from 'grommet/es6/components/Box'
 import { Button } from 'grommet/es6/components/Button'
 import { Heading } from 'grommet/es6/components/Heading'
+import { TextArea } from 'grommet/es6/components/TextArea'
 import { Text } from 'grommet/es6/components/Text'
 import { Copy } from 'grommet-icons/es6/icons/Copy'
-import { StatusWarning } from 'grommet-icons/es6/icons/StatusWarning'
+import { Dashboard } from 'grommet-icons/es6/icons/Dashboard'
 import * as React from 'react'
-import { useState } from 'react'
+import { useContext, useState } from 'react'
 import { Trans, useTranslation } from 'react-i18next'
 import { useSelector } from 'react-redux'
-
+import { normalizeColor } from 'grommet/es6/utils'
 import { ResponsiveLayer } from '../ResponsiveLayer'
+import logotype from '../../../../public/logo192.png'
+
+const StyledTextArea = styled(TextArea)`
+  // Opacity is a workaround for browsers anti-aliasing issue triggered by border-radius.
+  // Without it text is blurry.
+  background-color: ${({ theme }) => `${normalizeColor('brand-background-light', theme)}fe`};
```

Comment:
That's so weird!
It doesn't happen in Firefox, only Chrome. Did you find any issues about it?

Reply:
I did not look at any Chromium issue tracker yet.